cloudify_aws/s3/tests/test_bucket_object.py

```diff
@@ -44,7 +46,9 @@ def setUp(self):
         super(TestS3BucketObject, self).setUp()
         self.resource_config = RESOURCE_CONFIG
         self.resource_config['resource_config']['kwargs'] = \
-            {BUCKET: 'test_bucket'}
+            {
+                BUCKET: 'test_bucket'
+            }
```

Comment:
In my opinion, this should be ` = json.loads(json.dumps(my_dict))`. Essentially everything in the context is going to be JSON. And in JSON all strings are all unicode.

Reply:
Sure